inc/shortcodes/class-script.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Shortcake_Bakery\Shortcodes;
+
+class Script extends Shortcode {
+
+	public static function get_shortcode_ui_args() {
+		return array(
+			'label'          => esc_html__( 'Script', 'shortcake-bakery' ),
+			'listItemImage'  => 'dashicons-media-code',
+			'attrs'          => array(
+				array(
+					'label'        => esc_html__( 'URL', 'shortcake-bakery' ),
+					'attr'         => 'src',
+					'type'         => 'text',
+					'description'  => esc_html__( 'Full URL to the script file. Host must be whitelisted.', 'shortcake-bakery' ),
+				),
+			),
+		);
+	}
+
+	private static $whitelisted_script_domains = array();
+
+	/**
+	 * Get the whitelisted script domains for the plugin
```

Comment:
It would be good to have a little more explanation as to how the end-developer should use this.